docker-compose.yml

```diff
@@ -23,5 +23,26 @@ services:
       - APPLICATION_AUTHENTICATION_UI_ALLOWLIST=0.0.0.0/0
       - SPRING_H2_CONSOLE_SETTINGS_WEBALLOWOTHERS=true
 
+  community-api:
+    image: quay.io/hmpps/community-api:latest
+    container_name: community-api
+    ports:
+      - "9590:8080"
+      - "9592:9592"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
+    entrypoint: [
+        "java",
```

Comment:
as a matter of interest why do you need this entry pointy section? Is something missing from the community-api image?

Reply:
This was just to allow me to attach a remote debugger to Community API as I was unsure of a few bits around LAO.  Just forgot to remove it when committing.

Reply:
Have removed this now